app/assets/stylesheets/application.css.scss

```diff
@@ -106,19 +156,16 @@ h5 {
     padding: 1rem 0 0 1rem;
     margin: 0; }
 
-.row {
-  max-width: 1200px; }
-
 .footer {
   background-color: #191970;
   text-align: center;
   font-size: 1.2rem;
-  padding: 4rem; }
+  padding: 2rem; }
   .footer .inline-list a {
     display: inline-block;
     width: 2rem;
     height: 2rem;
-    background-color: #33383b;
+    background-color: #461B7E;
```

Comment:
Color literals like `#461B7E` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#461B7E` should be written as `#461b7e`